README.rst

```diff
@@ -155,8 +155,9 @@ intended you have no right to complain ;-).
    usage of the ``-H``, ``--hourly`` option for details about ``COUNT``."
    "``-t``, ``--timestamp-pattern=PATTERN``","Customize the regular expression pattern that is used to match and extract
    timestamps from filenames. ``PATTERN`` is expected to be a Python compatible
-   regular expression that must define the named capture groups 'year',
-   'month' and 'day' and may define 'hour', 'minute' and 'second'."
+   regular expression that must define a named capture group 'unixtime' or the
```

Comment:
@rsommer can you provide better documentations or an example of how to use this new unixtime capture group? I had to read the test case to understand it. There is a section lower in the docs containing some Regex examples, [Supported configuration options](https://github.com/xolox/python-rotate-backups#id7). That would be a good spot for more elaboration.

Reply:
I added a short example to the README file, hope that helps - otherwise this seems to be abandoned, since the original pull request was untouched for over a year :(